src/lib/components/List/List.jsx

```diff
@@ -12,32 +13,32 @@ export const List = ({
     return null;
   }
 
-  let alignClass = '';
+  let alignClass;
```

Comment:
Wait, I was setting the type here 🙂

Reply:
What do you mean? Before `classNames()`, you had had to explicitly pass empty string otherwise `[].join(' ')` would generate wrong classnames. Now, we can omit those empty string as undefined variable is correctly processed by `classNames()`.

Reply:
Just a joke, sorry 🙂. I mean that `''` was suggesting something like `let string: alignClass = ''`.